taskcluster/ci/toolchain/android.yml

```diff
@@ -30,3 +27,34 @@ linux64-android-sdk-linux-repack:
         toolchain-alias: android-sdk-linux
     treeherder:
         symbol: TL(android-sdk-linux)
+    worker:
+        docker-image: {in-tree: base}
+        max-run-time: 1800
+
+
+linux64-android-gradle-dependencies:
+    description: "Android Gradle dependencies toolchain task"
+    fetches:
+        toolchain:
+            # Aliases aren't allowed for toolchains depending on toolchains.
+            - linux64-android-sdk-linux-repack
+    run:
+        script: android-gradle-dependencies.sh
+        sparse-profile: null
+        resources:
+            - taskcluster/scripts/toolchain/android-gradle-dependencies.sh
+            - taskcluster/scripts/toolchain/android-gradle-dependencies/**
+            - buildSrc/src/main/java/Dependencies.kt
+            - '*.gradle'
```

Comment:
I expanded the list to include all components. 

⚠️ This task is spun if a single component gets its dependencies updated. This introduces a 2-hour-delay before getting results. I think it's okay if dependencies aren't updated that often, but it becomes a problem if many PRs do it. Does this sound okay-ish to you @pocmo?

Reply:
2 hours is harsh. At what point would we have to wait that long, when opening the PR? Or when merging it? Or in both cases?

What was the reason for including `build.gradle`?

Reply:
Great, after a lot of trial and error, I managed to reduce the task time down to 20-30 minutes, which looks more acceptable to me.

This task is rebuilt when a PR changes once of the listed files then when we merge it. The reason is: the cache built in a PR is considered untrusted. It becomes trusted once it's merged and once it runs on trusted workers. 

As far as I see, `build.gradle` lists the dependencies each component has (the main `build.gradle` also lists some) so we need to track these files to make sure we cache the latest dependencies.